addons/common/XEH_postInit.sqf

```diff
@@ -54,7 +54,9 @@
             _object setVariable [QGVAR(oldVisibility), _vis];
             _object setUnitTrait ["camouflageCoef", 0];
             {
-                _x forgetTarget _object;
+                if (side _x != side group _object) then {
```

Comment:
this will not work if `side _object == ENEMY`

Reply:
We actually just discussed this internally, it's quite edge case. One fix would just be to do:
```
if (side _x != side _object)
```

Reply:
The other would be to just have friendlies forget the unit too (with the exception of their own group for healing).